akvo/rsr/models/tests/test_project.py

```diff
@@ -0,0 +1,49 @@
+from akvo.rsr.models import Project
+from akvo.rsr.tests.base import BaseTestCase
+
+
+class EUTFHierarchyTest(BaseTestCase):
+    def test_not_eutf_root_project(self):
+        self.assertFalse(
+            Project.objects.create(title="Non EUTF").in_eutf_hierarchy()
+        )
+
+    def test_is_eutf_root_project(self):
+        eutf_project = Project.objects.create(title="Non EUTF")
```

Comment:
The project title is misleading.

Reply:
Fixed in 1bbbf0d094e95767be0d8b63c9fe94fa924bd769

Reply:
Fixed in 1bbbf0d094e95767be0d8b63c9fe94fa924bd769